website/docs/docs/use-dbt-semantic-layer/tableau.md

```diff
@@ -12,30 +12,33 @@ The Tableau integration with the dbt Semantic Layer is a [beta feature](/docs/db
 
 The Tableau integration allows you to use worksheets to query the Semantic Layer directly and produce your dashboards with trusted data.  
 
-This integration provides a live connection to the dbt Semantic Layer through Tableau Desktop.
+This integration provides a live connection to the dbt Semantic Layer through Tableau Desktop or Tableau Server. 
 
 ## Prerequisites
 
 - You have [configured the dbt Semantic Layer](/docs/use-dbt-semantic-layer/setup-sl) and are using dbt v1.6 or higher.
-- You must have [Tableau Desktop](https://www.tableau.com/en-gb/products/desktop) installed with version 2021.1 or greater
-   - Note that Tableau Online does not currently support custom connectors natively.
-- Log in to Tableau Desktop using either your license or the login details you use for Tableau Server or Tableau Online.
+- You must have [Tableau Desktop](https://www.tableau.com/en-gb/products/desktop) version 2021.1 and greater or Tableau Server.
+  - Note that Tableau Online does not currently support custom connectors natively. If you use Tableau Online, you will only be able to access the connector in Tableau Desktop.
+- Log in to Tableau Desktop (with Online or Server credentials) or a license to Tableau Server
 - You need your dbt Cloud host, [Environment ID](/docs/use-dbt-semantic-layer/setup-sl#set-up-dbt-semantic-layer) and [service token](/docs/dbt-cloud-apis/service-tokens) to log in. This account should be set up with the dbt Semantic Layer.
 - You must have a dbt Cloud Team or Enterprise [account](https://www.getdbt.com/pricing) and multi-tenant [deployment](/docs/cloud/about-cloud/regions-ip-addresses). (Single-Tenant coming soon)
 
 
-## Installing
+## Installing the Connector
 
 1. Download the GitHub [connector file](https://github.com/dbt-labs/semantic-layer-tableau-connector/releases/download/v1.0.2/dbt_semantic_layer.taco) locally and add it to your default folder:
-   - Windows: `C:\Users\\[Windows User]\Documents\My Tableau Repository\Connectors`
-   - Mac: `/Users/[user]/Documents/My Tableau Repository/Connectors`
-   - Linux: `/opt/tableau/connectors`
+
+|OS|Tableau Desktop|Tableau Server|
```

Comment:
@mirnawong1  clearly very bad at markdown tables because I can't get this to format! do you think you can look and see how to resolve?